<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>connector</class>
 <widget class="QDialog" name="connector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>134</width>
    <height>157</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>134</width>
    <height>157</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>134</width>
    <height>157</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="shaderImportBTN">
     <property name="text">
      <string>Shader import</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="applyShaderBTN">
     <property name="text">
      <string>Apply shader</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="applyAttrBTN">
     <property name="text">
      <string>Apply attributes</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
